docs/user/custom-resources/apirule/04-10-apirule-custom-resource.md

```diff
@@ -42,6 +42,7 @@ This table lists all parameters of APIRule CRD together with their descriptions:
 > [!WARNING]
 >  Having both the Oathkeeper and Istio `jwt` access strategies defined is not supported. Access
 > strategies `noop`, `allow`, or `no_auth` cannot be used with any other access strategy on the same **spec.rules.path**.
+> Mixing secure and unsecure access strategies is not supported.
```

Comment:
```suggestion
On the same **spec.rules.path**, you must not:
  - use the access strategies `jwt`, `oath2_introspection`, `noop`, `allow`, or `no_auth` with any other access strategy
  - define both the Oathkeeper and Istio `jwt` access strategies
  - combine secure and unsecure access strategies. The secure access strategies include `jwt` and `oath2_introspection`.
```
By 'secure access strategies' we mean `jwt` and `oath2_introspection`, right? Are there any cases when we do support defining more than one access strategy?

Reply:
> use the access strategies `jwt`, `oath2_introspection`, `noop`, `allow`, or `no_auth` with any other access strategy.

This is not correct, you can use jwt and oauth2_introspection in the same rule.
For the Ory handler "noop", "no_auth" and "allow" can't be used with any other access strategy.
For istio handler "jwt", "noop", "no_auth" and "allow" can't be used with any other access strategy.

Reply:
```suggestion
> When you use the Ory handler, do not define the access strategies `noop`, `allow`, or `no_auth` with any other access strategy on the same **spec.rules.path**.
> When you use the Istio handler, do not define the access strategies `jwt`, `noop`, `allow`, or `no_auth` with any other access strategy on the same **spec.rules.path**..
```
Thanks, would this be fine?

Reply:
I see. I edited the second suggestion and included the new validation

Reply:
Docs updated